rules/generate.rb

```diff
@@ -233,12 +240,20 @@ def write_html(file, sections)
    puts lines
    if file then
       File.open(file, "w") { |f|
-         f.puts(PHP_HEADER)
+         if plain then
+          f.puts(PLAINHTML_HEADER)
+         else
+          f.puts(PHP_HEADER)
+         end
          lines.each {|l| f.puts(l.gsub(/\(:/, '<span class="change">').
                                   gsub(/:\)/, '</span>').
                                   gsub(/\[\[([a-zA-Z0-9_-]*)\]\]/) { labels[$1] }.
-                                  gsub(/\{[^}]*\}/, '')) }
-         f.puts(PHP_FOOTER)
+                                  gsub(/\{([^}]*)\}/) { "<a name=\"" + $1 + "\"/>" } ) }
```

Comment:
@nwf It seems this line is causing the PHP version to wrap each section in an `<a>`, which is causing some interesting styling on hover. Propose making this conditional.

Reply:
Fixed a bit better in 8e012f8c4755bf82a78d7c79c6c370c183866c0e.